frontend/assets/stylesheets/components/_bundle-thank-you.scss

```diff
@@ -62,7 +62,12 @@
     }
 
     input[type=email] {
```

Comment:
Minor point, but should this be `type="email"`? Is that why the GitHub linter is shouting and making everything red?